.snapcraft/daemon.start

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -eu
+for i in "${SNAP_DATA}/nsd_server.pem" "${SNAP_DATA}/nsd_control.key" "${SNAP_DATA}/nsd_control.pem"; do
+    if ! [ -e "${i}" ]; then
+        echo "Missing certificate file: ${i}"
+        snapctl stop nsd-exporter.nsd-exporter
+        exit 0
+    fi
+done
+
+export GODEBUG="x509ignoreCN=0"
+exec nsd_exporter \
+    -listen-address :9167 \
```

Comment:
I'm not familiar with snap, how do you document to the user that the default listen address is different on snap?